tests/Auth0.AuthenticationApi.IntegrationTests/Tokens/IdTokenValidatorTests.cs

```diff
@@ -61,31 +61,45 @@ public async void ThrowsWhenIdTokenCanNotBeDecoded()
         }
 
         [Fact]
```

Comment:
Please add a test to cover the `none` algorithm case

Reply:
That test already exists here https://github.com/auth0/auth0.net/blob/master/tests/Auth0.AuthenticationApi.IntegrationTests/Tokens/IdTokenValidatorTests.cs#L93